src/vs/editor/contrib/codeAction/browser/codeActionMenu.ts

```diff
@@ -139,18 +156,49 @@ class CodeMenuRenderer implements IListRenderer<ICodeActionMenuItem, ICodeAction
 			data.root.classList.add('group-header');
 		} else {
 			const text = element.action.label;
-			data.text.textContent = text;
 			element.isEnabled = element.action.enabled;
 
 			if (element.action instanceof CodeActionAction) {
+				const openedFromString = (element.params?.options.fromLightbulb) ? CodeActionTriggerSource.Lightbulb : element.params?.trigger.triggerAction;
+
 
 				// Check documentation type
 				element.isDocumentation = element.action.action.kind === CodeActionMenu.documentationID;
 
 				if (element.isDocumentation) {
-					data.text.textContent = text;
+					element.isEnabled = false;
 					data.root.classList.add('documentation');
+
+					const container = data.root;
+
+					const actionbarContainer = dom.append(container, dom.$('.codeActionWidget-action-bar'));
+
+					const reRenderAction = showDisabled ?
+						<IAction>{
+							id: 'hideMoreCodeActions',
+							label: 'Hide More',
```

Comment:
The labels need to be localized

Also, instead of `More` I think it should make it clear that we are showing/hiding disabled actions. Maybe `Hide disabled` and `Show disabled`?